src/lib/updating-element.ts

```diff
@@ -579,24 +579,29 @@ export abstract class UpdatingElement extends HTMLElement {
    */
   requestUpdate(name?: PropertyKey, oldValue?: unknown) {
     let shouldRequestUpdate = true;
-    // if we have a property key, perform property update steps.
-    if (name !== undefined && !this._changedProperties.has(name)) {
+    // If we have a property key, perform property update steps.
+    if (name !== undefined) {
       const ctor = this.constructor as typeof UpdatingElement;
       const options =
           ctor._classProperties!.get(name) || defaultPropertyDeclaration;
       if (ctor._valueHasChanged(
```

Comment:
Unrelated to PR, but why do we pass `options.hasChanged` through a static method that just calls it?  It's private so it can't be overridden by user...

Reply:
I believe we intend to make this overridable at some point.